File: eco-core/core-plugin/src/main/java/com/willfp/ecoweapons/commands/CommandEcoweapons.java
package com.willfp.ecoweapons.commands;


import com.willfp.eco.core.EcoPlugin;
import com.willfp.eco.core.command.CommandHandler;
import com.willfp.eco.core.command.impl.PluginCommand;
import org.jetbrains.annotations.NotNull;

public class CommandEcoweapons extends PluginCommand {
    /**
     * Instantiate a new command handler.
     *
     * @param plugin The plugin for the commands to listen for.
     */
    public CommandEcoweapons(@NotNull final EcoPlugin plugin) {
        super(plugin, "ecoweapons", "ecoweapons.command.ecoweapons", false);

        this.addSubcommand(new CommandReload(plugin))
                .addSubcommand(new CommandGive(plugin));
    }

    @Override
    public CommandHandler getHandler() {
        return (sender, args) -> {
            sender.sendMessage(this.getPlugin().getLangYml().getMessage("invalid-command"));
        };
    }
}

File: eco-core/core-plugin/src/main/kotlin/com/willfp/ecoweapons/commands/CommandEcoweapons.kt
package com.willfp.ecoweapons.commands

import com.willfp.eco.core.EcoPlugin
import com.willfp.eco.core.command.CommandHandler
import com.willfp.eco.core.command.impl.PluginCommand

class CommandEcoweapons(
    plugin: EcoPlugin
) : PluginCommand(
        plugin,
        "ecoweapons",
        "ecoweapons.command.ecoweapons",
        false) {
    init {
        addSubcommand(CommandReload(plugin))
            .addSubcommand(CommandGive(plugin))
    }
    override fun getHandler(): CommandHandler {
        return CommandHandler { sender, _ ->
            sender.sendMessage(
                plugin.langYml.getMessage("invalid-command")
            )
        }
    }
}
